src/styles/blocks/calendar.scss

```diff
@@ -1,41 +1,38 @@
 .calendar {
-  padding: $calendar-padding;
-
-  &__container {
-    display: flex;
-    flex-wrap: wrap;
-    max-width: $week-size;
-    gap: $gap-between-days;
-    padding: $calendar-padding;
-    font-size: 30px;
 
-    @each $day, $value in $days {
-      &--start-day--#{$day} :first-child {
-        margin-left: $value * $width-gap-and-cell;
-      }
-    }
-  }
+  display: flex;
+  flex-wrap: wrap;
+  max-width: $week-size;
+  gap: $gap-between-days;
+  padding: $calendar-padding;
+  font-size: $font-size;
 
   @for $index from 28 through 31 {
-    &__container--month-length--#{$index} {
+    &--month-length-#{$index} {
       :nth-child(n + #{$index + 1}) {
         display: none;
       }
     }
   }
 
+  @each $day, $value in $days {
+    &--start-day-#{$day} :first-child {
+      margin-left: $value * $width-gap-and-cell;
+    }
+  }
+
   &__day {
     display: flex;
     align-items: center;
     justify-content: center;
     @include square($day-cell-size);
     border: 1px black solid;
-    background-color: #eee;
+    background-color: $bgc-of-dates;
     transition: background-color 0.5s, transform 0.5s;
 
     &:hover {
-      background-color: rgb(255, 0, 140);
-      color: rgb(255, 255, 255);
+      background-color: $bgc-hovered-dates;
+      color: $hovered-numbers-color;
       cursor: pointer;
```

Comment:
Move this prop to element itself.

Reply:
I don't understand, what it means. Can you pls explain?)